src/muya/lib/parser/index.js

```diff
@@ -182,6 +183,23 @@ const tokenizerFac = (src, beginRules, inlineRules, pos = 0, top, labels) => {
     if (inChunk) continue
     // image
     const imageTo = inlineRules.image.exec(src)
+    if (imageTo && typeof imageTo[4] === 'string') {
+      const lastParenIndex = findClosingBracket(imageTo[4], '()')
```

Comment:
typo

Reply:
Sorry, I didn't find the typo take place?

Reply:
I guess `lastParen<t>Index ` and not `lastParenIndex `?

Reply:
lastParenIndex => `last + paren + index`,  I think it's right? paren also means bracket.